lenses_python/SqlHandler.py

```diff
@@ -1,13 +1,15 @@
 from requests import *
 import pandas as pd
 from json import loads
-import sseclient
+from urllib.parse import urlencode
+import websocket
+
 
 from lenses_python.ConvertDateTime import ConvertDateTime
 
 class SqlHandler:
 
-    def __init__(self, url, username, password, token, query, datetimelist, formatinglist):
+    def __init__(self, url, username, password, token, query="", datetimelist=[], formatinglist=[]):
```

Comment:
why empty string and empty list in default params instead of None.

Reply:
the datetimelist and formatinglist must be list of strings which I use them in _ConvertToDF .

Reply:
yes that's fine but you will do this only when they are not none but ok. Empty list is mostly fine, empty string is weird, this should be `None`.